WordPress/Docs/WP/EnqueuedResourceParametersStandard.xml

```diff
@@ -0,0 +1,88 @@
+<documentation title="Enqueued Resource Parameters">
+    <standard>
+    <![CDATA[
+    The resource version must be set, to prevent the browser from retrieving an outdated, cached version after the resource has been updated.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Resource has a version number.">
+        <![CDATA[
+wp_register_script( 
+    'someScript-js', 
+    'https://example.com/someScript.js' , 
+    array( 'jquery' ),
+    <em>'1.0.0'</em>,
+    true
+);
+        ]]>
+        </code>
+        <code title="Invalid: Resource has no version number set.">
+        <![CDATA[
+wp_register_script( 
+    'someScript-js', 
+    'https://example.com/someScript.js' , 
+    array( 'jquery' ),
+    <em>null</em>,
```

Comment:
I'm wondering if the example would be clearer if the last two (optional) parameters in this _invalid_ example were not passed at all ?

I realize that it would make it awkward, or rather impossible for the `<em>` tags to be used in this example, but so be it.

Reply:
I've removed the null, indeed seems clearer this way, and the `<em>` is already in the valid example. Should be clear enough.